internal/services/notification/runwebhooksheduler.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Sorint.lab
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notification
+
+import (
+	"context"
+	"time"
+
+	"github.com/sorintlab/errors"
+
+	"agola.io/agola/internal/sqlg/lock"
+	"agola.io/agola/internal/sqlg/sql"
+	"agola.io/agola/services/notification/types"
+)
+
+const (
+	MaxRunWebhookLimit = 40
+	RunWebhookLockKey  = "runwebhook"
+)
+
+func (n *NotificationService) RunWebhookShedulerHandlerLoop(ctx context.Context) {
+	for {
+		if err := n.runWebhookShedulerHandler(ctx); err != nil {
+			n.log.Err(err).Send()
+		}
+
+		sleepCh := time.NewTimer(60 * time.Second).C
```

Comment:
Why 60 seconds?

Reply:
what is a good interval value? we want to add it in the configuration?